google-cloud-core/src/main/java/com/google/cloud/testing/BaseEmulatorHelper.java

```diff
@@ -337,9 +337,8 @@ public void start() throws IOException {
     }
 
     @Override
-    public void stop() throws InterruptedException {
+    public void waitFor() throws InterruptedException {
       if (process != null) {
-        process.destroy();
         process.waitFor();
```

Comment:
This puts us at the mercy of the emulator exiting. I'd like to see a timeout so that we don't wait forever if the emulator ignores us or locks up. Unfortunately, it might be a tiny bit of a pain to do this in Java 7:

http://stackoverflow.com/questions/808276/how-to-add-a-timeout-value-when-using-javas-runtime-exec